storage/container_storage_interface/persistent-storage-csi-migration.adoc

```diff
@@ -0,0 +1,59 @@
+[id="persistent-storage-csi-migration"]
+= CSI Automatic Migration
+
+toc::[]
+
+Starting with {product-title} 4.8, automatic migration for the following in-tree volume plug-ins to their equivalent Container Storage Interface (CSI) drivers is available:
+
+* Amazon Web Services (AWS) Elastic Block Storage (EBS)
+
+* OpenStack Cinder
+
+:FeatureName: CSI Automatic Migration
+include::modules/technology-preview.adoc[leveloffset=+1]
+
+With automatic migration enabled, volumes provisioned using in-tree storage plug-ins supported by this feature are migrated to their counterpart CSI drivers. By default, automatic migration is disabled.
+
+.Procedure
+
+You can enable and disable CSI automatic migration using feature gates (see xref:../../nodes/clusters/nodes-cluster-enabling-features.adoc[Enabling features using feature gates]).
+
+.Automatic migration to all in-scope CSI drivers:
+
+The following configuration example enables CSI automatic migration to both CSI drivers (EBS and Cinder):
```

Comment:
This should be a new task. It's debatable whether these should be completely separate modules. IMO, you can make it part of one procedure and detail perhaps like this:

.Procedure

To enable migration for all CSI drivers supported by CSI automatic migration:
-	(step shown above)

To enable CSI migration for a specific CSI driver: 
- 	Add the following `featureSet` reference by using the `CustomNoUpgrade` parameter and specifying the desired CSI driver you want to enable, as shown in the following example:

(Trying to think of a good way to present the parameter values for supported drivers. I'll leave a comment in the list of driver names below.)